prime-router/src/main/kotlin/Report.kt

```diff
@@ -253,7 +253,13 @@ class Report {
     fun split(): List<Report> {
         return itemIndices.map {
             val row = getRow(it)
-            Report(schema, values = listOf(row), sources = fromThisReport("split"), destination = destination)
+            Report(
+                schema = schema,
+                values = listOf(row),
+                sources = fromThisReport("split"),
+                destination = destination,
+                bodyFormat = bodyFormat
+            )
```

Comment:
Although not directly tied to the bug Jim found, this is a related bug that I found. Here `bodyFormat` is set to match the source report's format.